functions/model/teams/tark/addLabel.js

```diff
@@ -0,0 +1,44 @@
+const {getTeam, setLabelProperties, updateTeamDetails} =require("../lib");
```

Comment:
The eslint issues are not fixed here and can cause deployement issue, please fix them.